src/ai/backend/manager/api/auth.py

```diff
@@ -992,6 +992,34 @@ async def refresh_ssh_keypair(request: web.Request) -> web.Response:
     return web.json_response(data, status=200)
 
 
+@auth_required
+@check_api_params(
+    t.Dict(
+        {
+            t.Key("pubkey"): t.String,
+            t.Key("privkey"): t.String,
+        }
+    )
+)
+async def save_ssh_keypair(request: web.Request, params: Any) -> web.Response:
```

Comment:
How about renaming this method to `upload_ssh_keypair` and `refresh_ssh_keypair` to `generate_ssh_keypair` to make it clear?

Reply:
Great suggestion. I'll change to `upload_ssh_keypair` and `generate_ssh_keypair`

Reply:
I just pushed renamings.